plugins/callback/logstash.py

```diff
@@ -1,57 +1,86 @@
-# (C) 2016, Ievgen Khmelenko <ujenmr@gmail.com>
+# (C) 2020, Yevhen Khmelenko <ujenmr@gmail.com>
 # (C) 2017 Ansible Project
 # GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
 
 from __future__ import (absolute_import, division, print_function)
 __metaclass__ = type
 
-DOCUMENTATION = '''
-    author: Unknown (!UNKNOWN)
+DOCUMENTATION = r'''
     callback: logstash
     type: notification
+    author: Yevhen Khmelenko <ujenmr@gmail.com>
     short_description: Sends events to Logstash
     description:
       - This callback will report facts and task events to Logstash https://www.elastic.co/products/logstash
+    version_added: "2.3"
     requirements:
       - whitelisting in configuration
       - logstash (python library)
     options:
       server:
         description: Address of the Logstash server
-        env:
-          - name: LOGSTASH_SERVER
         ini:
           - section: callback_logstash
             key: server
-            version_added: 1.0.0
```

Comment:
This version_added needs to stay.

Reply:
added with https://github.com/ansible-collections/community.general/pull/641/commits/f32be815f0cb7f37bec0e72d752339d7fa709792

Reply:
The version_added is for the `ini` key, not for the option.

Reply:
like this https://github.com/ansible-collections/community.general/pull/641/commits/4d5a0e7c341162b378f60b1aec820541c42ff40e ?

Reply:
Yes, but now it's gone again. Please make sure that the diff for this PR doesn't remove anything for options you are not adding. Reordering keys is ok (though why have `env` after `ini`?), but not removing things.